configuration/roles.d/appkernels.json

```diff
@@ -0,0 +1,61 @@
+{
+    "roles": {
+        "po": {
```

Comment:
There is no `po` role in Open XDMoD.

Reply:
*facepalm* yes, these changes should be moved to ccr-private && then this file reverted to grant everybody access.